SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Microsoft.AppCenter.Crashes.Android.csproj

```diff
@@ -48,14 +48,12 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="PlatformCrashes.cs" />
+    <Compile Include="Crashes.cs" />
```

Comment:
Refactoring to using the static partial pattern like in core/push/distribute (analytics still need to align too).